apps/studio/src/components/Datatable/Datatable.tsx

```diff
@@ -136,18 +136,11 @@ export const Datatable = <T extends object>({
         </Table>
       </Box>
       <Flex py="1rem" gap="1rem">
-        {totalRowCountString && (
```

Comment:
Should keep right

Reply:
it wasn't in the design - see [here](https://www.figma.com/design/g7LnLn4IUcUC9dYxgsHEJM/IsomerCMS-V2-(working-title%3A-%22Isomer-Studio%22)?node-id=7336-40934&t=k3MIjEPJKGyVheCA-4)